### PR TITLE
LifeHash Central Repo

### PR DESCRIPTION
**Pull request by shannona**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,163 @@
...
+
+## Origin, Authors, Copyright & Licenses
+
+Unless otherwise noted (either in this [/README.md](./README.md) or in the file's header comments) the contents of this repository are Copyright © 2020-2022 by Blockchain Commons, LLC, and are [licensed](./LICENSE.md) under the [spdx:BSD-2-Clause Plus Patent License](https://spdx.org/licenses/BSD-2-Clause-Patent.html).
+
+In most cases, the authors, copyright, and license for each file reside in header comments in the source code. When it does not, we have attempted to attribute it accurately in the table below.
+
+## Financial Support
+
+**LIfeHash** is a project of [Blockchain Commons](https://www.blockchaincommons.com/). We are proudly a "not-for-profit" social benefit corporation committed to open source & open development. Our work is funded entirely by donations and collaborative partnerships with people like you. Every contribution will be spent on building open tools, technologies, and techniques that sustain and advance blockchain and internet security infrastructure and promote an open web.
```
Comment (ChristopherA) [inline README.md]: Typo “LIfehash”

**Review by ChristopherA — APPROVED** [review]
One minor typo spotted. Otherwise approved.

**Comment by shannona** [comment]
Thanks; always good to have those extra eyes. Fixed.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,163 @@
...
 
-This page is a demonstration of running the [LifeHash](https://github.com/BlockchainCommons/bc-lifehash) visual hash algorithm in the web browser. It uses the reference C++ implementation compiled to [WebAssembly](https://webassembly.org/). The page itself can be accessed [here](http://lifehash.info).
+* <img src="https://github.com/BlockchainCommons/crypto-commons/blob/master/images/logos/crypto-commons-super-simple.png" width=16 valign="bottom">&nbsp;&nbsp; ***part of the [crypto commons](https://github.com/BlockchainCommons/crypto-commons/blob/master/README.md) technology family***
 
-## Displaying a Specific LifeHash
+### _by [Wolf McNally](https://www.github.com/wolfmcnally) and [Christopher Allen](https://www.github.com/ChristopherA)_
 
-You can access this page's URL with a query parameter to populate the string input field with a specific value. Example:
+<a href="https://lifehash.info"><img src="Nexus-Images/lifehash-info.png" align="right" width=400></a>
+This is a central repo for LifeHash, a method of visual hash visualization that creates beautiful, deterministic icons.
```
Comment (wolfmcnally) [inline README.md]: "Visual hash visualization" is redundant. It should either be "visual hashing" or "hash visualization."

File: README.md [hunk 3]
```diff
@@ -1,26 +1,163 @@
...
+<br><br>
 
-[http://lifehash.info?s=RBH-AOZ](http://lifehash.info?s=RBH-AOZ)
+It also contains the data for our [lifehash.info](https://lifehash.info) demonstration site for LifeHash, which uses our [reference C++ implementation of LifeHash](https://github.com/BlockchainCommons/bc-lifehash) compiled to [WebAssembly](https://webassembly.org/). The page itself can be accessed [here](http://lifehash.info).
```
Comment (wolfmcnally) [inline README.md]: "It also contains the code for..."

File: README.md [hunk 4]
```diff
@@ -1,26 +1,163 @@
...
+
+[![LifeHash Explainer on YouTube](Nexus-Images/title-card.jpeg)](https://www.youtube.com/watch?v=cu0K__KLxKo)
+
+See a LifeHash demo at [LifeHash.info](https://lifehash.info/).
+
+## Additional Information
+
+The following files contain more information about LifeHash:
+
+* [Use Cases](https://github.com/BlockchainCommons/lifehash.info/blob/shannona-lifehash-nexus/Nexus-Docs/Use-Cases.md) — Why use LifeHash?
```
Comment (wolfmcnally) [inline README.md]: Looks like this URL is not in minimal form, like the others used in the list.

File: README.md [hunk 5]
```diff
@@ -1,26 +1,163 @@
...
+
+### Requesting a Specific LifeHash
+
+You can use a `s=` query parameter in the URL to populate the string input field with a specific value. For example:
+
+[http://lifehash.info?s=RBH-AOZ](http://lifehash.info?s=RBH-AOZ)
+
+If you want to generate a LifeHash directly from a SHA-256 digest, you can use a `f=` query parameter in the URL:
+
+[http://lifehash.info?f=70b0db749f1cb026282693b236048cefa3395fc580e91b7c62666b0bdc97c0e3](http://lifehash.info?f=70b0db749f1cb026282693b236048cefa3395fc580e91b7c62666b0bdc97c0e3)
```
Comment (wolfmcnally) [inline README.md]: This should be `d=`, not `f=`.

File: Nexus-Docs/Versions.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+## LifeHashes In Five Flavors
+
+LifeHashes can be displayed in one of five ways (one of which is deprecated:
+
+* **`version1`** The Original. **DEPRECATED**
+* **`version2`** Bug fixes and CMYK-friendly.
```
Comment (wolfmcnally) [inline Nexus-Docs/Versions.md]: I would add that this is the "official" LifeHash version.

File: Nexus-Docs/Versions.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+* **`detailed`** Now with twice the resolution, and CMYK-friendly.
+* **`fiducial`** [Fiducials](https://en.wikipedia.org/wiki/Fiducial_marker#Fiducial_marker_sets) are symbols specifically optimized for recognition by machine vision algorithms like those in Apple's [ARKit](https://developer.apple.com/augmented-reality/arkit/). Now CMYK-friendly.
```
Comment (wolfmcnally) [inline Nexus-Docs/Versions.md:8]: The word "now" is unnecessary.

**Review by wolfmcnally — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shannona) [reply README.md]: Replaced with "hash visualization"

File: README.md (hunk 3 above)
Comment (shannona) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (shannona) [reply README.md]: Yep, changed.

File: README.md (hunk 5 above)
Comment (shannona) [reply README.md]: I'd wondered what the f= stood for! Fixed.

File: Nexus-Docs/Versions.md (hunk 6 above)
Comment (shannona) [reply Nexus-Docs/Versions.md]: Noted.

File: Nexus-Docs/Versions.md (hunk 7 above)
Comment (shannona) [reply Nexus-Docs/Versions.md:8]: Fixed.

**Comment by shannona** [comment]
Thanks for great comments @Wolf, should all be resolved now.

**Review by wolfmcnally — COMMENTED** [review]
LGTM

**Pull request merged**